webapp/pages/profile/_id/_slug.vue

```diff
@@ -64,20 +66,21 @@
               </client-only>
             </ds-flex-item>
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-29T16:16:28Z" title="Wednesday, January 29th 2020, 5:16:28 pm +01:00">Jan 29, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -66,16 +68,23 @@
           </ds-flex>
           <ds-space margin="small">
```


I know this is not your code but here's an idea:
- put the `v-if` from the line below on the `ds-space`
- maybe make it a `div` (unless you come up with something more meaningful! :smile:)
- give it a class name and add the `margin` with CSS
- then you can target the buttons with `.this-class > .base-button` :tada:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T20:09:10Z" title="Wednesday, January 29th 2020, 9:09:10 pm +01:00">Jan 29, 2020</time>_

---


I like this idea, I'll have a look at it tomorrow!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-30T09:45:14Z" title="Thursday, January 30th 2020, 10:45:14 am +01:00">Jan 30, 2020</time>_

---


I looked at alternatives to `div`, with the goal of reducing the amount we use them, but didn't find anything I thought fit, would love your feedback.
I looked at https://developer.mozilla.org/en-US/docs/Web/HTML/Element and tried to do some reading, for example https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i 
I thought for a minute maybe `<section>`, but then read this 
```
The section element is not a generic container element. When an element is needed only for styling purposes or as a convenience for scripting, authors are encouraged to use the div element instead. A general rule is that the section element is appropriate only if the element's contents would be listed explicitly in the document's outline
```
and decided against it... 

I took a look as well at `<template>`, since the docs say it's `is a mechanism for holding HTML that is not to be rendered immediately when a page is loaded but may be instantiated subsequently during runtime using JavaScript.` , but it messed up the styling, meaning I would need to add more styling to get it to work. Feedback?